data/data-pipeline/data_pipeline/etl/sources/census_acs/etl.py

```diff
@@ -186,19 +186,25 @@ def __init__(self):
             "B03002_003E",
             "B03003_001E",
             "B03003_003E",
+            "B02001_007E",  # "Some other race alone"
         ]
 
-        # Name output demographics fields.
-        self.BLACK_FIELD_NAME = "Black or African American alone"
-        self.AMERICAN_INDIAN_FIELD_NAME = (
-            "American Indian and Alaska Native alone"
-        )
-        self.ASIAN_FIELD_NAME = "Asian alone"
-        self.HAWAIIAN_FIELD_NAME = "Native Hawaiian and Other Pacific alone"
-        self.TWO_OR_MORE_RACES_FIELD_NAME = "Two or more races"
-        self.NON_HISPANIC_WHITE_FIELD_NAME = "Non-Hispanic White"
+        self.BLACK_FIELD_NAME = "Black or African American"
+        self.AMERICAN_INDIAN_FIELD_NAME = "American Indian / Alaska Native"
+        self.ASIAN_FIELD_NAME = "Asian"
+        self.HAWAIIAN_FIELD_NAME = "Native Hawaiian or Pacific"
+        self.TWO_OR_MORE_RACES_FIELD_NAME = "two or more races"
+        self.NON_HISPANIC_WHITE_FIELD_NAME = "White"
         self.HISPANIC_FIELD_NAME = "Hispanic or Latino"
+        # Note that `other` is lowercase because the whole field will show up in the download
+        # file as "Percent other races"
+        self.OTHER_RACE_FIELD_NAME = "other races"
```

Comment:
`other race` (I think "other races" seems a bit too much like a multiracial descriptor, but it's not

Reply:
Eh, this is purely my personal preference, I'm gonna withdraw it